src/sql/workbench/contrib/executionPlan/browser/azdataGraphView.ts

```diff
@@ -93,6 +106,10 @@ export class AzdataGraphView extends Disposable {
 				}
 			}
 		});
+
+		this._diagram.graph.addListener('tooltipShown', (sender, evt) => {
+			status(evt.properties.tooltip.textContent);
```

Comment:
This seems like a really brute force way of doing this - imagine if you were trying to use a screen reader with this and it started reading off an entire paragraph of text with table information and everything. There isn't any way to pause or go back to content without starting from the beginning.

You should go to the a11y office hours to ask whether this is an acceptable solution for the issue, because to me it seems like people with screen readers should be using the properties window for this instead - although it presents information in a slightly different manner than the tooltip.